components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -134,7 +134,7 @@ export default class Workspaces extends React.Component<WorkspacesProps, Workspa
                         <div className="px-6 py-3 flex justify-between space-x-2 text-gray-400 border-t border-gray-200 dark:border-gray-800 h-96">
                             <div className="flex flex-col items-center w-96 m-auto">
                                 <h3 className="text-center pb-3 text-gray-500 dark:text-gray-400">No Active Workspaces</h3>
-                                <div className="text-center pb-6 text-gray-500">Prefix any git repository URL with gitpod.io/# or create a new workspace for a recently used project. <a className="gp-link" href="https://www.gitpod.io/docs/getting-started/">Learn more</a></div>
+                                <div className="text-center pb-6 text-gray-500">Import your existing projects by prefixing any git repository URL with gitpod.io/# or create a new workspace for a recently used project. <a className="gp-link" href="https://www.gitpod.io/docs/getting-started/">Learn more</a></div>
```

Comment:
**issue**: This change could be confusing once we ship **Teams & Projects** next month[[1](https://www.notion.so/gitpod/All-Hands-233ae3b1377249f9a32ffa3bb953bbd4#b2dda343d24a4767bf9c155d5b207e11)] as this will not actually _import_ a _project_ but only use the repository as a context. This could get more confusing as we're planning to add workspaces under personal accounts, teams, and projects (see https://github.com/gitpod-io/gitpod/pull/5555#discussion_r715007021). ❗ 

**thought**: We could consider having a way to prompt users to add repositories as projects on a team or their personal account while using the prefixed URL before opening the workspace but still this could be available only if they have proper write permissions so this change wouldn't be applicable for everyone.

**suggestion**: I'd suggest to hold this back this back until we ship Teams & Projects, rephrase it, or close it to avoid any confusion. What do you think?

Cc @svenefftinge because [relevant diff](https://github.com/gitpod-io/gitpod/pull/3295/files#diff-4a5c8bb479a125673053c7e43fe340a3bb1ffb478115a0003ae497c5005a1dcdR134).

Reply:
yes: importing a project != starting a workspace on a repo